var/spack/repos/builtin/packages/gcc/package.py

```diff
@@ -148,6 +171,16 @@ class Gcc(AutotoolsPackage):
     # See https://gcc.gnu.org/gcc-5/changes.html
     conflicts('languages=jit', when='@:4')
 
+    # NVPTX offloading supported in 7 and later by limited languages
+    conflicts('+nvptx', when='@:6')
+    conflicts('languages=ada', when='+nvptx')
+    conflicts('languages=brig', when='+nvptx')
+    conflicts('languages=go', when='+nvptx')
+    conflicts('languages=java', when='+nvptx')
+    conflicts('languages=jit', when='+nvptx')
+    conflicts('languages=objc', when='+nvptx')
+    conflicts('languages=obj-c++', when='+nvptx')
```

Comment:
I'm a bit on the fence regarding these language conflicts. Gcc still builds when compiling it with e.g. `spack install gcc +nvptx languages=objc,c,c++,fortran`, and offloads C/C++/Fortran correctly. So these language conflicts might not be necessary.

Reply:
Ok, makes sense.